packages/tokens/src/global/radii.ts

```diff
@@ -0,0 +1,13 @@
+export interface LineHeightProps extends Array<number> {
+  sharp?: number;
+  xsmall?: number;
+  small?: number;
+  regular?: number;
+  circle?: number;
+}
+
+const radii: LineHeightProps = [0, 4, 8, 16, 9999];
+
+[radii.sharp, radii.xsmall, radii.small, radii.regular, radii.circle] = radii;
+
+export default radii;
```

Comment:
I believe you should do the same here regarding @MicaelRodrigues and my comments <3